Tigerhacker/fractured-bootstrapper
# Electron update to solve Let's Encrypt Cert expiration

Updated electron and friends

Tested:

- Log In (Only Manual Auth Works atm)
- Logout
- Launch Game
- Install Game Files
- Re-Install (After delete)
- Check Patch
